tests/expectations/compiler/futures/nested.out

```diff
@@ -1,18 +1,5 @@
 DCE_ENABLED:
-Warning [WSAZ0374002]: The type checker has exceeded the max depth of nested conditional blocks: 0.
```

Comment:
These errors should still be detected.

Reply:
My understanding was that we decided to remove `condition-branch-type-checking` but still keep `max-depth` to verify that users correctly await all of their futures.

Reply:
Ah you're right, I had incorrectly read the diff and though that the errors were being pulled out entirely.